src/_gettsim/synthetic.py

```diff
@@ -287,8 +306,6 @@ def create_constant_across_households_variables(df, n_adults, n_children, policy
             bg_daten["bruttokaltmiete"][hh_typ_string_lookup]
         ),
         "heizkosten_m_hh": float(bg_daten["heizkosten"][hh_typ_string_lookup]),
-        "kind_unterh_anspr_m": 0.0,
```

Comment:
Why did you remove this and the next line?

Reply:
all float input variables which are not defined in this dictionnary are set to 0.0 anyway below. So no need to include those two here.

Reply:
Ah thanks, didn't know that!